### ISSUE TITLE
[BUG] django.db.utils.OperationalError: no such table: documents_document

### ISSUE THREAD

**Issue by amo13** [jonaswinkler/paperless-ng#1106]
**Describe the bug**
When trying to import PDFs into a fresh installation of Paperless-ng on Archlinux, Paperless complains about a missing `documents_document` table in the database.
Any Idea of help with this issue is greatly appreciated.

**To Reproduce**
1. Clean installation of paperless-ng
2. Launch the webserver, consumer and scheduler
3. Drop a PDF onto the web interface or in the consume folder

**Expected behavior**
Process the imported PDF.

**Scheduler logs**
```
11:14:56 [Q] INFO Process-1:4 pushing tasks at 21426
11:14:56 [Q] INFO Process-1:1 processing [BRWD812655C0E62_000324.pdf]
11:14:56 [Q] INFO Q Cluster double-timing-missouri-twenty running.
11:14:56 [Q] INFO Process-1:3 monitoring at 21425
11:14:56 [Q] INFO Process-1:1 stopped doing work
11:14:56 [Q] ERROR no such table: django_q_task
11:14:56 [Q] ERROR Failed [BRWD812655C0E62_000324.pdf] - no such table: documents_document : Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 423, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: documents_document
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 430, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/share/paperless/src/documents/tasks.py", line 74, in consume_file
    document = Consumer().try_consume_file(
  File "/usr/share/paperless/src/documents/consumer.py", line 197, in try_consume_file
    self.pre_check_duplicate()
  File "/usr/share/paperless/src/documents/consumer.py", line 94, in pre_check_duplicate
    if Document.objects.filter(Q(checksum=checksum) | Q(archive_checksum=checksum)).exists():  # NOQA: E501
  File "/usr/lib/python3.9/site-packages/django/db/models/query.py", line 808, in exists
    return self.query.has_results(using=self.db)
  File "/usr/lib/python3.9/site-packages/django/db/models/sql/query.py", line 550, in has_results
    return compiler.has_results()
  File "/usr/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1145, in has_results
    return bool(self.execute_sql(SINGLE))
  File "/usr/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1175, in execute_sql
    cursor.execute(sql, params)
  File "/usr/lib/python3.9/site-packages/django/db/backends/utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/usr/lib/python3.9/site-packages/django/db/backends/utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/usr/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python3.9/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 423, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: documents_document
```

**Relevant information**
 - Host OS of the machine running paperless: Archlinux
 - Version 1.4.4
 - Installation method: bare metal
 - dependencies differing from pipfile: django-q (1.3.6 instead of 1.3.4), numpy (1.20.3 instead of 1.19.5), pillow (8.2.0 instead of 8.1), pikepdf (2.12.1 instead of 2.5), scipy (1.6.3 instead of 1.5.4), watchdog (0.10.6 instead of 1.0), inotifyrecursive (0.3.5 instead of 0.3.4), uvloop (0.15.1 instead of 0.14). Since the error seems to be dropped by django itself, I guess none of these dependencies are to blame.

**Comment by jonaswinkler** [comment]
Did you run the database migration steps?

`python manage.py migrate`

**Comment by amo13** [comment]
Thank you for your answer.
Yes, this is happening on an instance that has been running for a while. I tried to re-run `migrate` but it makes no difference.

**Comment by amo13** [comment]
A fresh installation of 1.4.5 does not show this error.

**Comment by amo13** [comment]
Ok, so the error showed up after an update. The update pulls the `paperless.conf` template in the paperless directory. 

What I did not expect is that this template `paperless.conf` takes precedence before `/etc/paperless.conf`. So I got this bug because my paperless instance was using the template config instead of mine. 

So I must ask: shouldn't either the template `paperless.conf` file be renamed to `paperless.conf.template` or the precedence be changed to first `/etc/paperless.conf` and only second `./paperless.conf`?
